telegram/ext/_dispatcher.py

```diff
@@ -573,11 +570,43 @@ def add_handler(self, handler: Handler[UT, CCT], group: int = DEFAULT_GROUP) ->
 
         if group not in self.handlers:
             self.handlers[group] = []
-            self.groups.append(group)
-            self.groups = sorted(self.groups)
 
         self.handlers[group].append(handler)
 
+    def add_handlers(
+        self,
+        handlers: Union[Sequence[Handler], Dict[int, Sequence[Handler]]],
+        group: int = DEFAULT_GROUP,
```

Comment:
Maybe using `DefaultValue` here is safer - that way we know for user that the user didn't pass `group = DEFAULT_GROUP`